locations/spiders/discover_swiss.py

```diff
@@ -0,0 +1,122 @@
+import urllib.parse
+
+import scrapy
+
+from locations.categories import Categories, PaymentMethods, apply_category, map_payment
+from locations.items import Feature
+
+
+# Hotels (including hostels and serviced apartments) in Switzerland
+class DiscoverSwissSpider(scrapy.Spider):
+    name = "discover_swiss"
+    allowed_domains = ["api.discover.swiss"]
+    dataset_attributes = {
+        # This dataset includes some data from other sources, like hotel
+        # chains. But a lot of their content is unique, such as small hotels
+        # that aren’t part of a chain.
+        "aggregate": "yes",
+        # Mandatory attribution as per CC-BY 4.0, waived for OpenStreetMap
+        # via standard template. Negotiations took place in January 2025
+        # between Hotellerie Suisse (who runs the discover.swiss platform)
+        # and the Swiss OpenStreetMap association.
+        # https://osmfoundation.org/wiki/Licence/Waiver_and_Permission_Templates
+        "attribution": "mandatory",
```

Comment:
```suggestion
        "attribution": "required",
```

To match the others.